### ISSUE TITLE
Text on auto reduction home does not meet WCAG guidelines

### ISSUE THREAD

**Issue by keiranjprice101** [ISISScientificComputing/autoreduce#663]
Issue raised by: [developer]

### What?
The footer text does not meet [WCAG](https://www.w3.org/TR/WCAG21/#contrast-enhanced) contrast guidelines. Currently the ratio is 1.6:1

![image](https://user-images.githubusercontent.com/44777678/87354431-750f0780-c556-11ea-9d50-2f556bbf3fa9.png)

This may be true for other text/logos in the webapp but I did not check.

### Where?
Being unfamiliar with the webapp, I don't know where to look to change this. @ElliotAOram I guess you would be able to point in the right direction.

**Comment by ElliotAOram** [comment]
Good consideration here. You can find the files of interest here:
html source: `WebApp/autoreduce_webapp/templates/footer.html`
css: `WebApp/autoreduce_webapp/static/css/base.css`
